examples/use-kits.yaml

```diff
@@ -0,0 +1,89 @@
+# Example: Use of non-serialized resources in quantity (ouikits)
+
+'@context':
+  - https://git.io/vf-examples-jsonld-context
+  - ouikit: http://www.ouikit.org/
+
+'@id': rgh:valueflows/valueflows/master/examples/use-kits.yaml
+'@graph':
+
+  # recipe
+
+  - '@id': ouikit:a7e94dde-2997-451e-a7e1-8d364d5e6dee
+    '@type': ResourceSpecification
+    name: Ouikits
+    skos:note: An eating kit containing plate, bowl, cup, utensils.
+    unit: unit:Number
+    substitutable: True
+
+  - '@id': ouikit:c404cff5-19c7-453d-b0d8-d8e96055bd0b
+    '@type': RecipeProcess
+    skos:note: Hold a workshop with meal
+    time:hasDuration:
+      time:unitType: time:unitHour
+      time:numericDuration: 4
+    scalingFormula: D
+
+  - '@id': ouikit:ff208521-9fec-4bb4-a6d3-197f6735cb26
+    '@type': RecipeFlow
+    action: use
+    resourceConformsTo: ouikit:a7e94dde-2997-451e-a7e1-8d364d5e6dee  # reusable eating kit
+    quantifiedAs:
+      qudt:unit: unit:Number
+      qudt:numericValue: 1
+    alsoQuantifiedAs:
+      qudt:unit: unit:Hour
+      qudt:numericValue: 1
+    scalingFormula: Q*M
+
+  # [..... many other inputs and outputs....]
+
+  # a resource
+
+  - '@id': ouikit:9396d2d1-ded3-4b1c-8144-1c024e641039
+    '@type': EconomicResource
+    resourceConformsTo: ouikit:a7e94dde-2997-451e-a7e1-8d364d5e6dee  # reusable eating kit
+    currentQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 100
+
+  # planning the event
+
+  - '@id': ouikit:a3277840-54b9-497e-9cf6-1902eda1584e
+    '@type': Process
+    name: Workshop dinner 2019-03-15
+    hasDuration:
+      time:unitType: time:unitHour
+      time:numericDuration: 4
+
+  - '@id': ouikit:bd5806d6-4a36-45b9-b3b6-3e7d361a5a13
+    '@type': Commitment
+    action: use
+    provider: http://www.ouikit.org/
+    receiver: http://www.ouikit.org/
+    involves: ouikit:9396d2d1-ded3-4b1c-8144-1c024e641039 # ouikits resource
+    quantifiedAs:
+      qudt:unit: unit:Number
+      qudt:numericValue: 30
+    hasBeginning: 2019-03-15T13:00:00-5:00
+    hasEnd: 2019-03-15T17:00:00-5:00
+
+  # [..... many other inputs and outputs....]
+
+  # using the kits for the event
+
+  - '@id': ouikit:30dad10d-606e-4188-80a2-4a1cf2454f58
+    '@type': EconomicEvent
+    action: use
+    provider: http://www.ouikit.org/
+    receiver: http://www.ouikit.org/
+    involves: ouikit:9396d2d1-ded3-4b1c-8144-1c024e641039 # ouikits resource
+    quantifiedAs:
+      qudt:unit: unit:Number
+      qudt:numericValue: 29
+    alsoQuantifiedAs:
+      qudt:unit: unit:Hour
+      qudt:numericValue: 2.5
```

Comment:
How application supposed to understand what `quantifiedAs` and `alsoQuantifiedAs` supposed to mean? Would anything change if we swap those two values?

Reply:
Yeah, quantifiedAs follows down from intents and commitments.  You could only fulfill a commitment if the event is using the same meaning of quantity and the same unit, compare planned vs actual, etc.  The alsoQuantifiedAs would be a different unit or at least a different functional quantity than found in intents or commitments before the event.

Reply:
> You could only fulfill a commitment if the event is using the same meaning of quantity and the same unit, compare planned vs actual, etc.

and where application supposed to get the 'meaning of quantity?

> The alsoQuantifiedAs would be a different unit or at least a different functional quantity than found in intents or commitments before the event.

I see no reason why Intents and Commitments wouldn't have both number of kits and usage duration and require satisfaction and fulfillment of both. So neither 10 kits for 2.5 hour or 29 for 1 hour can fully satisfy or fulfill.

Reply:
>I see no reason why Intents and Commitments wouldn't have both number of kits and usage duration and require satisfaction and fulfillment of both.

But why would they need that? The scheduling and coordination happens with the time fields.

Reply:
>and where application supposed to get the 'meaning of quantity?

Haven't really seen a problem in practice so far.  The resource specification knows the unit, so that clues people in.  And there is a description on a commitment that adds any detail not provided by the action.  If you see a form to fill in that says "work" "translate english to spanish" and "hours", you just fill in your hours and the time information.  In our UIs you can see on the same page the process information and the input document, etc.

For usage, you see the equipment, or at least its specification, the unit, so not confusing.  For consume and produce, the same.

I wasn't sure what to do with the taxi service stuff, but I don't actually like what I did in the current state of the example, I'm going to change it, but need to think a bit.